.vscode/cspell.json

```diff
@@ -199,14 +200,15 @@
     {
       "filename": "**/sdk/sqlmanagement/**/*.cs",
       "words": [
-        "Droppeded",
         "CIAS",
-        "Vcores",
-        "Unrestorable",
-        "Stdev",
+        "Droppeded",
+        "Nchar",
         "Ntext",
         "Nvarchar",
-        "Stdev"
+        "Stdev",
+        "Stdev",
```

Comment:
Are these duplicates?

Reply:
They are... More reason to sort this list whenever adding to it.